File: C/retangulo.c
#include <stdio.h>
#include <math.h>

int main() {
    
    double baseRetangulo, alturaRetangulo, area, perimetro, diagonal;
    
    printf("Base do retangulo: ");
    scanf("%lf", &baseRetangulo);
    
    printf("Altura do retangulo ");
    scanf("%lf", &alturaRetangulo);
    
    area = (baseRetangulo * alturaRetangulo);
    perimetro = ( 2 * (baseRetangulo + alturaRetangulo));
    diagonal = pow(baseRetangulo, 2) + pow(alturaRetangulo, 2);
    diagonal = sqrt(diagonal); 
    
    printf("AREA = %.4lf\n", area);
    printf("PERIMETRO = %.4lf\n", perimetro);
    printf("DIAGONAL = %.4lf", diagonal);
    
    return 0;
}

File: JAVA/retangulo.java
import java.util.Locale;
import java.util.Scanner;

public class retangulo {

	public static void main(String[] args) {
		
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		double valorBase, valorAltura, area, perimetro, diagonal;
		
		System.out.print("Base do retangulo: ");
		valorBase = sc.nextDouble();
		
		System.out.print("Altura do retangulo: ");
		valorAltura = sc.nextDouble();
		
		area = (valorBase * valorAltura);
		
		perimetro = ((valorBase * 2) + (valorAltura * 2));
		
		diagonal = Math.sqrt(Math.pow(valorBase, 2) + Math.pow(valorAltura, 2));
		
		System.out.printf("AREA = %.4f%n", area);

		System.out.printf("PERIMETRO = %.4f%n", perimetro);
		
		System.out.printf("DIAGONAL = %.4f%n", diagonal);
		
		sc.close();
		
	}

}
